docs/sdk/client-side-sdks/android/android-usage.md

```diff
@@ -280,7 +113,7 @@ Map<String, Feature<Object>> variables = dvcClient.allFeatures();
 If the SDK has not finished initializing, these methods will return an empty Map.
 
 
-### Get All Variables
+## Get All Variables
```

Comment:
do you know why for Get All Variables and Get All Features it just has **Kotlin** vs the other usages have Kotlin Example as a header? should we make it consistent?

Reply:
yes we should, will do!